web/src/app/shared/store/workbasket-store/workbasket.actions.ts

```diff
@@ -0,0 +1,102 @@
+import { Workbasket } from '../../models/workbasket';
+import { TaskanaQueryParameters } from '../../util/query-parameters';
+import { Direction } from '../../models/sorting';
+import { ACTION } from '../../models/action';
+import { WorkbasketAccessItems } from '../../models/workbasket-access-items';
+
+// Workbasket List
+export class GetWorkbasketsSummary {
+  static readonly type = '[Workbasket List] Get all workbaskets\' summary';
+
+  constructor(public forceRequest: boolean = false,
+    public sortBy: string = TaskanaQueryParameters.parameters.KEY,
+    public order: string = Direction.ASC,
```

Comment:
why does this not just use a Sorting and a Filter as parameters?
This would ease understandability and readability in my opinion.

Reply:
because this would means also changing the service file while I did not want to, may cause unexpected bugs